python/interpret_community/explanation/explanation.py

```diff
@@ -220,17 +220,20 @@ class FeatureImportanceExplanation(BaseExplanation):
     :type features: Union[list[str], list[int]]
     """
 
-    def __init__(self, features=None, is_raw=False, **kwargs):
+    def __init__(self, features=None, is_raw=False, is_eng=False, **kwargs):
         """Create the feature importance explanation from the given feature names.
 
         :param features: The feature names.
         :type features: Union[list[str], list[int]]
-        :param is_raw: Whether the explanation is on the original (raw) features or engineered ones.
+        :param is_raw: Whether the explanation is on the original (raw) features or not.
```

Comment:
nit: I think the or not can be removed: "Whether the explanation is on the original (raw) features."